### PR TITLE
Update Charter to reflect committer-member difference

### PR DESCRIPTION
**Pull request by achopragh**
Updating the charter to indicate the two formal roles of a TSC voting member and the project committer. Added 1b and 1-d-iv lines only.

### CODE REVIEW

File: CHARTER.md
Comment (nezihyigitbasi) [inline CHARTER.md:20]:
```suggestion
b. The TSC will consist of two roles - TSC voting members and Committers. 
```

File: CHARTER.md
Comment (nezihyigitbasi) [inline CHARTER.md:32]: `A Project Committer` -> `Committer` ?

Comment (nezihyigitbasi) [reply CHARTER.md:32]: `which will also require approval` -> `which will require approval`

**Review by nezihyigitbasi — APPROVED** [review]
This has been voted & approved by TSC. Please squash & merge. Thanks!

**Comment by achopragh** [comment]
The TSC has voted and approved the change.

**Pull request merged**
